### ISSUE TITLE
Python 3.9.12 vs pypy3.9-7.3.9 

### ISSUE THREAD

**Issue by dhutchin-ctl** [irrdnet/irrd#700]
**Describe the bug**
Using pipenv to manage my virtual environment, the installation of irrd fails when pypy is used instead of CPython. 

**To Reproduce**
Include 
[pypyfail.bash.gz](https://github.com/irrdnet/irrd/files/10059275/pypyfail.bash.gz)
The attached bash script run on CentOS 7.9 should work to reproduce the problem.  It installs a clean Python virtual environment and then runs a basic install of IRRd V4 via pipenv.
WARNING: Script removes .pyenv, .pki, .venv, .cache, .local, Pipfile, and Pipfile.local from the user's home directory. Save these off before running this script.  I don't believe the script generates anything outside of these file/directories.

**Expected behaviour**
When the above script is run with "component" set to '3.9.12' (line ~20) the script succeeds.  When component is set to pypy3.9-7.3.9 (which reports itself as Python 3.9.12), it fails.

**IRRd version you are running**
Running (ie installing) with default version from pypi.org (on 2022/11/21).  But I think other versions of irrd fail, too, needs more testing on my part.

**Additional context**
Last summer, this script worked with pypy3.9-7.3.9 and IRRd V4.2.5. Something in one of the dependencies changed.

**Comment by 2bithacker** [comment]
Tested this out in a clean Ubuntu Focal environment and ran into the same issue until I installed build-essential and libpq-dev. This is probably down to the Postgres library changing. Under CPython, IRRd requires psycopg2-binary, but under PyPy is uses psycopg2cffi instead, which needs the libpq libraries around to build.

Not sure what the CentOS equiv to libpq-dev is, but I suspect it needs to be installed.
